src/main/scala/io/qbeast/spark/internal/sources/v2/QbeastBatch.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Qbeast Analytics, S.L.
```

Comment:
I guess we should update the copyright to 2023 😅

Reply:
I will fix it, thank you.